docs/source/api/deliveryservices.rst

```diff
@@ -31,21 +31,30 @@ Request Structure
-	+-------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
-	| logsEnabled | no       | Show only the :term:`Delivery Services` that have :ref:`ds-logs-enabled` set or not based on this boolean                                  |
-	+-------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
-	| profile     | no       | Return only :term:`Delivery Services` using the :term:`Profile` that has this :ref:`profile-id`                                            |
-	+-------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
-	| tenant      | no       | Show only the :term:`Delivery Services` belonging to the :term:`Tenant` identified by this integral, unique identifier                     |
-	+-------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
-	| type        | no       | Return only :term:`Delivery Services` of the :term:`Delivery Service` :ref:`ds-types` identified by this integral, unique identifier       |
-	+-------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+	+-------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Name        | Required | Description                                                                                                                                    |
+	+=============+==========+================================================================================================================================================+
+	| cdn         | no       | Show only the :term:`Delivery Service`\ s belonging to the CDN identified by this integral, unique identifier                                  |
+	+-------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------+
+	| id          | no       | Show only the :term:`Delivery Service` that has this integral, unique identifier                                                               |
+	+-------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------+
+	| logsEnabled | no       | If true, return only :term:`Delivery Service`\ s with logging enabled, otherwise return only :term:`Delivery Service`\ s with logging disabled |
```

Comment:
This is probably from rebasing, but it looks like for most of these (on this page, that is) you don't have this text copied exactly from what it was.

Reply:
I'm confused now, which version is what I need to have?

Reply:
See how above - in the 'removed' section of the diff - it has links to the DS overview sections for things like 'logging enabled'; that stuff should persist, so it doesn't need to be explained everywhere.